lib/realtime_web/live/inspector_live/conn_component.html.heex

```diff
@@ -78,6 +78,20 @@
       <%= error_tag(f, :table) %>
       <p class="text-gray-600 text-xs italic">Listen to changes from this table</p>
     </div>
+    <div class="mb-4">
+      <%= label(f, :filter, class: "block text-gray-700 text-sm font-bold mb-2") %>
+      <%= text_input(f, :filter, class: "
```

Comment:
@filipecabaco i recommend adding a helpful placeholder, e.g. `body=eq.hey`, since the filter syntax can be tricky to remember.